src/ui/sass/modules/_AppliedFilters.scss

```diff
@@ -0,0 +1,84 @@
+/** @define AppliedFilter */
+
+.yxt-AppliedFilters
+{
+  margin-right: var(--yxt-base-spacing);
+  border-top: 0;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+
+  &-filterLabel
+  {
+    display: flex;
+    margin-right: calc(var(--yxt-base-spacing) / 4);
+    margin-bottom: 4px;
+    @include Text(
+      $size: var(--yxt-font-size-md),
+      $line-height: var(--yxt-line-height-md),
+      $color: var(--yxt-color-text-secondary),
+    );
+  }
+
+  &-filterValue
```

Comment:
Could this and &-filterLabel could be consolidated since they're mostly the same?

Reply:
updated!